input/pagecontent/future.md

```diff
@@ -29,4 +29,9 @@ The approach to develop and mature AU Core is outlined below making reference to
     <img src="maturing.png" alt="Maturing the AU Core baseline" style="width:80%"/>
   </div>
 *Figure 3: Maturing the AU Core baseline*
-<br/>
\ No newline at end of file
+<br/>
+
+### AU Core Profiles Roadmap 
```

Comment:
Heading should be "Future Candidate Requirements Under Consideration"
Have a sentence under the heading. "The following items are under consideration to add AU Core."

Reply:
Done